### DISCUSSION TITLE
Pet utilité

### DISCUSSION THREAD

**Discussion by DeadAngelV6** [Crownicles/Crownicles#573] [Idées acceptées / déjà implémentées]
Donner une utilité au pet, exemple : 
Donne + de points / xp;
Donne de la vie;
Aide en combat (attaque en + d'une attaque normal)
Donne un espace en + dans l'inventaire gratuitement ?

**Comment by BastLast** [comment]
Le mini event des pets va déjà être buff fortement à la prochaine mise à jour

**Comment by Pietagorh** [comment]
potentiellement distribuer les caractéristiques (pv, exp, points, fight, inv) à chaque pet ?

**Comment by niqore** [comment]
Et du coup augmenter le bonus en fonction du dressage (comme ça ça donne un but à le dresser). Genre s'il est 50% dressé, le bonus ne sera que de 50% du total

**Reply by Pietagorh** [reply]
Mais du coup comment tu implémentes ça pour les pets qui ont des trucs d'inv en plus ? Peut-être un cooldown plus ou moins élevé pour accéder à l'objet qui est dedans ?

**Reply by beaud37210** [reply]
En vrai c'est une bonne idée

**Comment by BastLast** [comment]
du coup mtn les familiers peuvent participer aux combats et partir en expedition donc je ferme cette suggestion comme acceptée
